### ISSUE TITLE
Release rjulia via CRAN?

### ISSUE THREAD

**Issue by phaverty** [armgong/rjulia#48]
What do you think about adding rjulia to the CRAN repository of packages?  I've got some work in the BioConductor repository that could make good use of rjulia. Doing so would require having rjulia in BioConductor or CRAN, though.  CRAN might be nice way to increase the visibility of your work.

**Comment by mmaechler** [comment]
I agree.   Getting this to CRAN had been one of my initial motivations for getting involved here.
It may need some work (configuration / checking of Julia's availability/...) that you should talk with the CRAN team about how to do "properly" ... well in way that makes sense and does not give too much extra work for the team.

**Comment by alexpghayes** [comment]
Curious if there's been any movement on this front? I'm considering `rjulia` for a project and am hoping to make things as user friendly as possible.

In a similar vein, would it be possible to have a function `rjulia::install_julia()` like [blogdown::install_hugo()](https://github.com/rstudio/blogdown/blob/master/R/install.R)?

**Comment by phaverty** [comment]
I'm afraid there hasn't been any work towards this goal. The suggested example function looks like a good template to follow.  I'm still interested in making this happen. Knowing others are interested too makes it much more likely!

**Comment by alexpghayes** [comment]
I don't know enough about Julia to do much, but I can contribute to code cleanup/trynna get CMD check to pass.

At the moment, I can't even install `rjulia` on windows, but once I figure out to get that going I'm down to get a move on. Definitely keep me in the loop.
